lib/ansible/plugins/inventory/scaleway.py

```diff
@@ -27,6 +27,12 @@
         tags:
             description: Filter results on a specific tag
             type: list
+        mandatory_tags:
```

Comment:
Why are you adding `mandatory_tags` when you already have `tags`? Wouldn't it be better to add a notion of conditions in filters? Like to be able to search a device with a tag A *and* tag B?